test/e2e/data/e2e_eks_conf.yaml

```diff
@@ -136,7 +136,7 @@ variables:
 
 intervals:
   default/wait-cluster: ["30m", "10s"]
-  default/wait-control-plane: ["30m", "10s"]
+  default/wait-control-plane: ["45m", "10s"]
```

Comment:
This is a big increase, which if required makes my a bit uneasy as it indicates a big change in behaviour somewhere. But lets looks after a successful run :)

Reply:
Running ci artifacts conformance test with reduced timeout.

Reply:
Works fine with 35min timeout.